internal/service/vpc/resource_gcp_vpc_peering_connection.go

```diff
@@ -206,9 +206,17 @@ func resourceGCPVPCPeeringConnectionImport(ctx context.Context, d *schema.Resour
 		return nil, fmt.Errorf("invalid identifier %v, expected <project_name>/<vpc_id>/<gcp_project_id>/<peer_vpc>", d.Id())
 	}
 
+	client := m.(*aiven.Client)
+
+	projectName, vpcID, peerCloudAccount, peerVPC, peerRegion := parsePeeringVPCId(d.Id())
```

Comment:
GCP has a different ID format, without `peerRegion`

Reply:
it returns `*string` and will be `nil` there, which won't be passed to the API